### PR TITLE
Improve geonode dockerization for master branch

### PR DESCRIPTION
**Pull request by Vampouille**

### CODE REVIEW

**Comment by Vampouille** [comment]
Since 2.7.x branch I have following error when trying to connect as admin : 

```
NoReverseMatch at /search/

Reverse for 'account_invite_user' with arguments '()' and keyword arguments '{}' not found. 0 pattern(s) tried: []
```
@francbartoli Do you known how to fix this issue ?

**Comment by francbartoli** [comment]
@Vampouille regarding the issue I think it should have been fixed in master. Maybe @afabiani can confirm.

**Comment by Vampouille** [comment]
> I think it should have been fixed in master.

This PR is based on master, it seems that this issue is still present on master.

File: docker-compose.yml [hunk 1]
```diff
@@ -1,49 +1,87 @@
 version: '2'
+
 services:
 
-  postgres:
-    image: postgres
+  database:
+    image: camptocamp/postgis:9.5
```
Comment (francbartoli) [inline docker-compose.yml]: @Vampouille can we rely on official image here?

Comment (Vampouille) [reply docker-compose.yml]: There is no official postgis image :
https://store.docker.com/search?q=postgis&source=verified&type=image
and we need postgis extension for geoserver database.

**Comment by Vampouille** [comment]
travis failed with `ServerNotFoundError: Unable to find the server at geoserver`

File: docker-compose.yml (hunk 1 above)
Comment (francbartoli) [reply docker-compose.yml]: Ok then I'll go to create a geonode official through docker hub by forking your repo

**Comment by francbartoli** [comment]
@Vampouille I see travis failed. Did you run successfully the test on your local machine or elsewhere?

**Comment by Vampouille** [comment]
On local with venv, running :
`./manage.py test geonode.people.tests geonode.base.tests geonode.layers.tests geonode.maps.tests geonode.proxy.test
s geonode.security.tests geonode.social.tests geonode.catalogue.tests geonode.documents.tests geonode.api.tests geonode.groups.tests geonode.services.tests geonode.geoserver.tests geonode.up
load.tests geonode.tasks.tests --noinput --failfast`
failed with : 
```
ERROR: testLayerNotifications (geonode.layers.tests.LayerNotificationsTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/jacroute/git/geonode/labs_geonode/geonode/layers/tests.py", line 1069, in testLayerNotifications
    self.assertTrue(self.check_notification_out('layer_updated', self.u))
  File "/home/jacroute/git/geonode/labs_geonode/geonode/tests/utils.py", line 189, in check_notification_out
    user.noticesetting_set.get(notice_type__label=notification_name)
  File "/home/jacroute/git/geonode/labs_geonode/venv/local/lib/python2.7/site-packages/django/db/models/manager.py", line 127, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/jacroute/git/geonode/labs_geonode/venv/local/lib/python2.7/site-packages/django/db/models/query.py", line 328, in get
    num = len(clone)
  File "/home/jacroute/git/geonode/labs_geonode/venv/local/lib/python2.7/site-packages/django/db/models/query.py", line 144, in __len__
    self._fetch_all()
  File "/home/jacroute/git/geonode/labs_geonode/venv/local/lib/python2.7/site-packages/django/db/models/query.py", line 965, in _fetch_all
    self._result_cache = list(self.iterator())
  File "/home/jacroute/git/geonode/labs_geonode/venv/local/lib/python2.7/site-packages/django/db/models/query.py", line 238, in iterator
    results = compiler.execute_sql()
  File "/home/jacroute/git/geonode/labs_geonode/venv/local/lib/python2.7/site-packages/django/db/models/sql/compiler.py", line 840, in execute_sql
    cursor.execute(sql, params)
  File "/home/jacroute/git/geonode/labs_geonode/venv/local/lib/python2.7/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/home/jacroute/git/geonode/labs_geonode/venv/local/lib/python2.7/site-packages/django/db/utils.py", line 98, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/home/jacroute/git/geonode/labs_geonode/venv/local/lib/python2.7/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/home/jacroute/git/geonode/labs_geonode/venv/local/lib/python2.7/site-packages/django/db/backends/sqlite3/base.py", line 318, in execute
    return Database.Cursor.execute(self, query, params)
OperationalError: no such table: notification_noticesetting
```

**Comment by Vampouille** [comment]
Running `./manage.py test` :
```
Internal Server Error: /documents/upload
...
NoReverseMatch: Reverse for 'account_invite_user' with arguments '()' and keyword arguments '{}' not found. 0 pattern(s) tried: []
E...EEEEE..............EEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE..........EEEEEEEEEInternal Server Error: /services/register/
```
Maybe I missing some dependencies ?

**Comment by francbartoli** [comment]
@Vampouille I'm going to close this in favor of #3538 which is now more or less stable. I'm happy if you can test it

**Pull request closed without merging**